ZConfig/components/logger/handlers.xml

```diff
@@ -12,46 +12,126 @@
   </sectiontype>
 
   <sectiontype name="logfile"
                datatype=".FileHandlerFactory"
                implements="ZConfig.logger.handler"
                extends="ZConfig.logger.base-log-handler">
-	<example><![CDATA[
+    <example><![CDATA[
         <logfile>
             path STDOUT
             format %(name)s %(message)s
         </logfile>
     ]]>
     </example>
-    <key name="path" required="yes"/>
-    <key name="old-files" required="no" default="0" datatype="integer"/>
-    <key name="max-size" required="no" default="0" datatype="byte-size"/>
-    <key name="when" required="no" default="" datatype="string"/>
-    <key name="interval" required="no" default="0" datatype="integer"/>
+    <key name="path" required="yes">
+      <description>
+        Path of the log file to write.
+
+        Specifying ``STDOUT`` or ``STDERR`` will cause the appropriate
+        standard stream to be used instead of a file.  In these cases,
+        rotation, encoding, and delayed opening are not available, and
+        will be considered configuration errors.
+      </description>
+    </key>
+    <key name="old-files" required="no" default="0" datatype="integer">
+      <description format="rest">
+        Number of old log files which will be retained when rotation is
+        configured.
+
+        This must be set to a positive integer if rotation is
+        configured.  An error is generated if rotation is requested
+        without setting ``old-files``.
+      </description>
+    </key>
+    <key name="max-size" required="no" default="0" datatype="byte-size">
+      <description>
+        Target maximum size for a logfile; once a logfile reaches the
+        maximum size, it will be rotated.
+      </description>
+    </key>
+    <key name="when" required="no" default="" datatype="string">
+      <description format="rest">
+        Specification for specific time at which rotation should occur.
+
+        Allowed values are described for
+        :class:`logging.handlers.TimedRotatingFileHandler`; this value is
+        passed through to the underlying handler.
+      </description>
+    </key>
+    <key name="interval" required="no" default="0" datatype="integer">
+      <description format="rest">
+        Frequency of rotation at the time specified by ``when``.
+
+        If not specified, but ``when`` is specified, this will default
+        to ``1``.
+
+        If ``when`` is not specified, it is an error to specify
+        ``interval``.
+      </description>
+    </key>
     <key name="format"
          default="------\n%(asctime)s %(levelname)s %(name)s %(message)s"
-         datatype=".log_format"/>
+         datatype=".log_format">
+      <description format="rest">
+        Format string for log entries.  This value is used to create an
+        instance of the class identified by ``formatter``.
+
+        The following escape characters are supported with the same
+        replacements as in Python string literals:
```

Comment:
Is it two colons `::` for a pre-formatted block? I thought simple indentation was a quoted block.

Reply:
Thanks for catching this!  I clearly reviewed the formatted HTML too quickly.

Fixed in commit 5f85187.